packages/admin-ui/src/lib/static/styles/global/_sass-overrides.scss

```diff
@@ -0,0 +1,3 @@
+// Note: variables defined in this file should use the default! flag so they can be
+// overridden by custom Admin UI applications
+$clr-sidenav-width: 10.8rem !default;
```

Comment:
This is a late addition. Using the `default!` flag for Vendure overrides allows the developer to override this value on their Admin UI app - which I need, actually.